File: array/easy/largest_element.java
public class  main{
    static int check_largest(int[] arr){
        int largest = arr[0];
        for(int i = 0; i<arr.length; i++){
            if(arr[i] > largest){
                largest = arr[i];
            }
        }
        return largest;
    }
    public static void main(String[] args){
    int[] arr = {1,2,3,4,5};
    System.out.print(check_largest(arr));
    }
}

File: array_problem_revise/largest_element.js
function findTheLargest(arr){
    let largest = arr[0];
    for(let i = 0; i < arr.length; i++){
        if(arr[i]  >  largest){
            largest = arr[i];
        }
    }
    return largest;
}
console.log(findTheLargest([1,2,3,4,5]))
 


let arr  = [5,2,9,5,3,4];
console.log(Math.max(...arr));

